tests/functionaltests/test_end_to_end_smoke.py

```diff
@@ -66,11 +57,11 @@ def read_first_blob_from_container(context):
     blob_data = blob_client.download_blob().readall()
     content = blob_data.decode('utf-8')  # For text files
 
-    print(f"First blob name: {first_blob_name}")
-    print("Content:")
-    print(content)
+    data=json.loads(content)
     payload_path = os.path.join(os.path.dirname(__file__), 'payloads/sample_payload.json')
     with open(payload_path, 'r') as f:
         payload = json.load(f)
-    assert json.dumps(payload) == content
+
+    # First record from the uploaded blob file compared with payload (event)
+    assert payload == data[0]
```

Comment:
Does this bit need clarifying? There's only one record in [payloads/sample_payload.json](https://github.com/NHSDigital/dtos-analyse-data-pipeline/blob/4abfc8a45b69126a60b9a3a4d020e6ad9e3d17f5/tests/functionaltests/payloads/sample_payload.json)

So, should it be something like:

```suggestion
    assert payload == data[0], "File content from blob storage should match payload posted"
```

Reply:
@gjackson296 this has been addressed now and relevant code has been modified